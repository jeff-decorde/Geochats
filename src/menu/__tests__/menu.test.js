import React from 'react';
import Menu from '../menu.js';

describe('Menu', () => {
  const wrapper = shallow(<Menu />);
  
  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot(wrapper);
  })
})
