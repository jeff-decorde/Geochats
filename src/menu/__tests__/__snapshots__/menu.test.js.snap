// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[object Object] 1`] = `
<nav
  aria-label="main navigation"
  className="navbar"
>
  <div
    className="navbar-brand"
  >
    <a
      className="navbar-item"
      href="https://artechglobal.com.au/"
    >
      <img
        alt="Geochats"
        height="28"
        src="https://artechglobal.com.au/wp-content/uploads/2018/05/ARtech-Logo.png"
        width="112"
      />
    </a>
  </div>
</nav>
`;
